From 37e5494f2b350c1069201d91a55a1cc755f1de06 Mon Sep 17 00:00:00 2001
From: Yuhang Zhao <2546789017@qq.com>
Date: Mon, 4 Jul 2022 17:42:36 +0800
Subject: [PATCH] wangwenx190: Disable debug configuration for release builds

Signed-off-by: Yuhang Zhao <2546789017@qq.com>
---
 ports/ffmpeg/portfile.cmake | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/ports/ffmpeg/portfile.cmake b/ports/ffmpeg/portfile.cmake
index f18677776..26e38da87 100644
--- a/ports/ffmpeg/portfile.cmake
+++ b/ports/ffmpeg/portfile.cmake
@@ -52,7 +52,7 @@ else()
     set(LIB_PATH_VAR "LIBRARY_PATH")
 endif()
 
-set(OPTIONS "--enable-pic --disable-doc --enable-debug --enable-runtime-cpudetect")
+set(OPTIONS "--enable-pic --disable-doc --enable-runtime-cpudetect")
 
 if(VCPKG_TARGET_ARCHITECTURE STREQUAL "arm")
   set(OPTIONS "${OPTIONS} --disable-asm --disable-x86asm")
@@ -480,8 +480,8 @@ if(VCPKG_TARGET_IS_UWP)
 endif()
 
 # Note: --disable-optimizations can't be used due to https://ffmpeg.org/pipermail/libav-user/2013-March/003945.html
-set(OPTIONS_DEBUG "--debug")
-set(OPTIONS_RELEASE "")
+set(OPTIONS_DEBUG "--enable-debug --debug")
+set(OPTIONS_RELEASE "--disable-debug")
 
 set(OPTIONS "${OPTIONS} ${OPTIONS_CROSS}")
 
-- 
2.39.0.windows.1

